RELEASING.md

```diff
@@ -0,0 +1,41 @@
+# Releasing
+
+The following artifacts are created as a result of releasing Fabric Chaincode Node:
+
+- docker images
+    - [fabric-nodeenv](https://hub.docker.com/r/hyperledger/fabric-nodeenv)
+- npm modules
+    - [fabric-contract-api](https://www.npmjs.com/package/fabric-contract-api)
+    - [fabric-shim](https://www.npmjs.com/package/fabric-shim)
+    - [fabric-shim-api](https://www.npmjs.com/package/fabric-shim-api)
+    - [fabric-shim-crypto](https://www.npmjs.com/package/fabric-shim-crypto)
+
+**Note:** A docker image with a matching V.R version is required before releasing a new version of Fabric.
+
+## Before releasing
+
+It's useful to create an issue to keep track of each release, for example [FABCN-377 Release v2.0.0 chaincode-node](https://jira.hyperledger.org/browse/FABCN-377).
+
+The following tasks are normally required before releasing, in addition to any specific to a particular release:
+
+- Update version numbers in package.json files to the required version
+- Update `tag` in package.json files to the required value, e.g. `beta`, or `latest`
+- Update test, sample, and docs files to match the new version
+- Create a new release notes file
+- Update the `CHANGELOG.md` file
+  
+  The `changelog.sh` script in `tools/scripts` will prepopulate the changelog but you must check and edit the file manually afterwards as required
+
+See the [Prepare 2.1.4 release](https://github.com/hyperledger/fabric-chaincode-node/pull/174) pull request for an example, although be careful to search for all versions in the codebase as they're easy to miss and things change!
+
+## Create release
+
+Creating a GitHub release on the [releases page](https://github.com/hyperledger/fabric-chaincode-node/releases) will trigger the build to publish the new release.
```

Comment:
Might want to include some comment about the name of the tag and what to put in the text.